wdl/pipelines/TechAgnostic/TertiaryAnalysis/ProcessMalariaBarcodesDemo.wdl

```diff
@@ -0,0 +1,292 @@
+version 1.0
+
+##########################################################################################
```

Comment:
move the description to the meta block of the workflow

Reply:
Fixed!